dashboards/grafana-dashboard-kruize-metrics.configmap.yaml

```
@@ -4494,8 +3916,8 @@ data:
       },
       "timezone": "",
       "title": "Kruize Metrics",
-      "uid": "playground",
-      "version": 1,
+      "uid": "kruizemetricsUx9248",
+      "version": 8,
```

```suggestion
      "version": 1,
```

Is this dashboard available in in stage/PROD instance?
In case not then could you please keep version as `1`? Once available, for next change one can update version if required.